pkgs/fc/ceph/src/fc/ceph/tests/test_maintenance.py

```diff
@@ -32,9 +32,19 @@ def ceph_json_calls(monkeypatch):
     return ceph_json_calls
 
 
-# FIXME: so far only testing nautilus behaviour
-def test_successful_maintenance_cycle(ceph_json_calls, locktoolcalls):
-    maintenance_task = fc.ceph.maintenance.nautilus.MaintenanceTasks()
+def test_successful_maintenance_cycle(
+    ceph_json_calls, locktoolcalls, maintenance_manager
+):
+    # only relevant from Nautilus on
```

Comment:
# only relevant from Nautilus on
     if maintenance_manager.__name__ in ["jewel", "luminous"]:
        return